hs_core/hydroshare/hs_bagit.py

```diff
@@ -197,6 +197,12 @@ def create_bag(resource):
     # set bag_modified-true AVU pair for on-demand bagging.to indicate the resource bag needs to be created when user clicks on download button
     istorage.setAVU(resource.short_id, "bag_modified", "true")
 
+    # set isPublic metadata AVU isPublic accordingly
+    istorage.setAVU(resource.short_id, "isPublic", str(resource.raccess.public))
+
+    # set resourceType metadata AVU resourceType accordingly
```

Comment:
These comments are not adding any new information and are superfluous. You could roll these `istorage` statements into one block to read nicer.